src/AcceptanceTests/TestSuiteConstraints.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using AcceptanceTesting.Support;
+
+    public partial class TestSuiteConstraints
+    {
+        public bool SupportsDtc { get; } = false;
+        public bool SupportsCrossQueueTransactions { get; } = true; // TODO: verify that this is an AtomicSendWithReceive. If not, set to false
```

Comment:
@danielmarbach so this flag means, cross queue tx support in receive only mode? we should set it to false then. And I wonder why the related test is green then... asb can't do this, it needs atomic sends with receive mode